template/engine.go

```diff
@@ -98,13 +98,13 @@ func (e *Engine) ParseTemplates() error {
 }
 
 // Render process a template.
-func (e *Engine) Render(name, language string, data interface{}) []byte {
+func (e *Engine) Render(name string, data map[string]interface{}) []byte {
```

Comment:
I'm not very familiar with go, I think making this more specific should be fine? Please advice. I wanted avoid long type assertion below.